ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -105,7 +105,10 @@ def verify_image_versions(old_images, upgrade_version):
         old_images, selector=constants.CSI_RBDPLUGIN_PROVISIONER_LABEL,
         count=2
     )
-    verify_pods_upgraded(old_images, selector=constants.MON_APP_LABEL, count=3)
+    # Increased timeout for BZ https://bugzilla.redhat.com/show_bug.cgi?id=1856254
+    verify_pods_upgraded(
```

Comment:
with merge of this PR: https://github.com/red-hat-storage/ocs-ci/pull/2453, this part moved to ocs_upgrade.py